pkg/pillar/cmd/volumemgr/handledownloader.go

```diff
@@ -204,6 +205,7 @@ func handleDownloaderStatusDelete(ctxArg interface{}, key string,
 	ctx := ctxArg.(*volumemgrContext)
 	status := statusArg.(types.DownloaderStatus)
 	updateVolumeStatus(ctx, status.ObjType, status.ImageSha256, status.ImageID)
+	updateContentTreeStatus(ctx, status.ImageSha256, status.ImageID)
```

Comment:
Shouldn't you decide whether it is a volume or CT and call the appropriate update function?

Reply:
We are updating the VolumeStatus and ContentTreeStatus based on the Sha in the config/status.

Reply:
We don't need a separate function to update the content tree status. In latest changes, we are updating both VolumeStatus and ContentTreeStatus in the same function.